Framework/src/InfrastructureGenerator.cxx

```diff
@@ -47,6 +47,9 @@ using SubSpec = o2::header::DataHeader::SubSpecificationType;
 namespace o2::quality_control::core
 {
 
+const char* defaultRemotePort = "36543";
```

Comment:
why not string?

Reply:
no reason. I will change it to string if the checks fail anyway, otherwise I wouldn't block this with running the checks again.

Reply:
indeed